parser/src/parsercontext.cpp

```diff
@@ -0,0 +1,73 @@
+#include <parser/parsercontext.h>
+#include <parser/sourcemanager.h>
+
+namespace po = boost::program_options;
+
+namespace cc
+{
+namespace parser
+{
+
+ParserContext::ParserContext(
+    std::shared_ptr<odb::database> db_,
+    SourceManager& srcMgr_,
+    po::variables_map& options_,
+    std::unordered_map<std::string, IncrementalStatus> fileStatus_):
+    db(db_),
```

Comment:
Indent the initialisers two more spaces.

Reply:
Fixed in eaae9fdac3faadd46558e484a28a7930a32d3591.